### PR TITLE
Update attributions in various locations

### PR DESCRIPTION
**Pull request by mia-pi-git**

### CODE REVIEW

File: server/chat-plugins/help.ts [hunk 1]
```diff
@@ -3,7 +3,8 @@
  * Uses Regex to match room frequently asked question (RFAQ) entries,
  * and replies if a match is found.
  * Supports configuration.
- * Written by mia-pi.
+ * Written by Mia.
+ * @author mia-pi-git
```
Comment (Zarel) [inline server/chat-plugins/help.ts:7]: There's no need to use your Git username for `@author`, I think you can just use whatever you want.

Comment (Zarel) [reply server/chat-plugins/help.ts:7]: https://jsdoc.app/tags-author.html

Comment (mia-pi-git) [reply server/chat-plugins/help.ts:7]: Yes, i'm just picky about how I have it.

**Comment by Zarel** [comment]
Looks like we forgot about this; thanks for the bump.

**Pull request merged**